### PR TITLE
YALB- 988: Override Gin colors and typography

### PR DESCRIPTION
**Pull request by mariannuar**
### Description of work
- Override gin colors and typography.

### Functional testing steps:
- [x] Visit any page: https://pr-254-yalesites-platform.pantheonsite.io/ (login)
- [x] Confirm the left sidebard looks as expected in the [designs](https://www.figma.com/file/CrE2jOK5R1BYZJ0EzfGnHr/YaleSites-AX-UI-Designs?node-id=448-353&t=gXyWIiYjDoy8CpfD-4)
- [x] Go to layout
- [x] Confirm it looks as expected in the design (Modals are not included in this PR)

### Notes
- I have to injected a css file in the atomic theme because when you are viewing or editing the layout of a node, the css styles added in the `ys_admin_theme` didn't override the gin colors and typography. I tried to find other places, like in the `ys_toolbar`, but it didn't work.
- I thought there would be a preview for this, so I don't know how Ashley can review this part of the work.
- Review this [PR](https://github.com/yalesites-org/yalesites-project/pull/254) too, please.

### CODE REVIEW

**Comment by joetower** [comment]
@codechefmarc Could you also give this a look when you have time, please? Main PR is here: https://github.com/yalesites-org/yalesites-project/pull/254

File: css/admin-theme.css [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  --gin-bg-app: #353535;
+  --gin-color-text-light: var(--light-gray);
+  --gin-color-primary-light-active: var(--robin-egg);
+  --gin-icon-color: var(--light-gray);
+  --gin-bg-layer2: #fff;
+  --gin-bg-layer3: #fff;
+  --gin-color-button-text: #fff;
+}
+
+:root h1,
```
Comment (codechefmarc) [inline css/admin-theme.css:50]: I understand the need to include the admin-css on the front-end, however, I think this h1 is leaking through and causing the page title to become more bolded than it used to be.
Screenshot 2023-05-03 at 9 27 39 AM

**Comment by joetower** [comment]
@codechefmarc This is ready for a retest, please. 

https://pr-254-yalesites-platform.pantheonsite.io

**Review by codechefmarc — APPROVED** [review]
Works great for me. Also noticed that the modals are styled in layout builder even though someplace you mentioned modals weren't included, they work for me. Approved!

**Pull request merged**